app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+  <p><%= contestant.name %></p>
+  <% contestant.projects.each do |project| %>
+    <p><%= project.name %></p>
+  <% end %>
+<% end %>
```

Comment:
For testing purposes, making sure that the correct projects are under the correct ocntestantants, you might want to use a unique class to target specific sections of this page.